src/nav/integrations.yml

```diff
@@ -249,7 +249,7 @@ pages:
             path: /docs/integrations/amazon-integrations/troubleshooting/no-data-appears-aws-integrations
           - title: Throttled AWS data
             path: /docs/integrations/amazon-integrations/aws-integrations-list/rate-limit-alerts-amazon
-          - title: '&quot;Invalid principal&quot; error'
+          - title: '"Invalid principal&quot; error'
```

Comment:
Looks like you'll want to do a regex global replace to ensure it captures both quotes